src/schema.rs

```diff
@@ -0,0 +1 @@
+infer_schema!("dotenv:DATABASE_URL");
```

Comment:
I think there is another way to do this that is probably better. The way this currently works is by reading your *local running database* and using that to *compile this application*. That is a dependency you do not want for an actual product...

There should be some way to specify your scheme through sql files -- please use that instead